canonical-web-and-design/buy.ubuntu.com
# Add GTM

## Done

- Added GTM to theme.liquid and checkout.liquid
- Removed livechat include, GTM injects this now

## QA
- Visit https://canonical-staging.myshopify.com
- Inspect the source code, see that the GTM scripts are there
- In the GTM dashboard, verify that it's working

Fixes https://github.com/canonical-web-and-design/buy.ubuntu.com/issues/48
